styles/styles.css

```
@@ -1,279 +1,423 @@
-    line-height: 26px;
-    font-weight: 700;
+.filter-hotel-title {
+  font-size: 20px;
+  line-height: 26px;
+  font-weight: 700;
 }
 
-.button-catalog{
-    background-color: #83B3D3;
+.button-catalog {
+  background-color: #83b3d3;
 }
 
-.ghost{ 
-    background-color: rgba(117, 190, 218, 0.0);
-    border: none;
-    color: #ffffff;
+.ghost {
+  background-color: rgba(117, 190, 218, 0);
+  border: none;
+  color: #ffffff;
 }
 
-.sorting-results{
-    background-color: #ffffff;
-    color: #333333;
-    font-size: 18px;
-    line-height: 23px;
+.sorting-results {
+  background-color: #ffffff;
+  color: #333333;
+  font-size: 18px;
+  line-height: 23px;
 }
 
-.results-title{
-    font-size: 30px;
-    line-height: 39px;
-    font-weight: 700;
-    color: #000000;
+.results-title {
+  font-size: 30px;
+  line-height: 39px;
+  font-weight: 700;
+  color: #000000;
 }
 
-.form-show-item{
-    list-style: none;
+.form-show-item {
+  list-style: none;
 }
 
-.form-show-link{
-    color: #000000;
-    text-decoration: none;
+.form-show-link {
+  color: #000000;
+  text-decoration: none;
 }
 
-.search-results-title{
-    font-size: 24px;
-    line-height: 31px;
-    font-weight: 700;
+.search-results-title {
+  font-size: 24px;
+  line-height: 31px;
+  font-weight: 700;
 }
 
-.rating{
-    font-size: 16px;
-    line-height: 21px;
+.rating {
+  font-size: 16px;
+  line-height: 21px;
 }
 
-.results-buttom{
-    background-color: #756257;
+.results-buttom {
+  background-color: #756257;
 }
 
-.footer-results{
-    font-size: 20px;
-    line-height: 26px;
+.footer-results {
+  font-size: 20px;
+  line-height: 26px;
 }
 
-.download-button{
-    font-size: 20px;
-    line-height: 26px;
+.download-button {
+  font-size: 20px;
+  line-height: 26px;
+}
+
+.paginator-link {
+  color: #ffffff;
+  text-decoration: none;
+}
+
+.paginator-item {
+  list-style: none;
+  background-color: #83b3d3;
+  font-weight: 700;
+}
+
+.acrive-page {
+  color: #000000;
+  background-color: #f2f2f2;
+}
+
+.acrive-page a {
+  color: #000000;
+}
+
+.paginator-button {
+  font-size: 20px;
+  line-height: 26px;
 }
 
-.paginator-link{
-    color: #FFFFFF;
-     text-decoration: none;
+.subscription-button {
+  font-size: 20px;
+  line-height: 26px;
 }
 
-.paginator-item{
-    list-style: none;
-    background-color:  #83B3D3;
-    font-weight: 700;
+
+.header-container {
+  width: 1060px;
+  margin: 0 auto;
 }
 
-.acrive-page{
-    color: #000000;
-    background-color: #F2F2F2;
+.navigation {
+  display: flex;
 }
 
-.acrive-page a{
-    color: #000000;
+.navigation-list {
+  margin: 0;
+  padding: 0;
+  width: 440px;
 }
 
-.paginator-button{
-    font-size: 20px;
-    line-height: 26px;
+.navigation-user {
+  margin-left: auto;
+  width: 300px;
 }
 
-.subscription-button{
-    font-size: 20px;
-    line-height: 26px;
+.logo-header {
+  display: block;
+  margin-right: 30px;
 }
 
+.logo-intro {
+  display: block;
+  margin: 0 auto;
+}
 
+.benefits-list, .city-sedona-list {
```

По стайлгайду, перечисление с новой строки
```suggestion
.benefits-list, 
.city-sedona-list {
```